markdown/extensions/md_in_html.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, md: Markdown, *args, **kwargs):
         self.block_tags = set(self.block_level_tags) - (self.span_tags | self.raw_tags | self.empty_tags)
         self.span_and_blocks_tags = self.block_tags | self.span_tags
 
-    def reset(self):
+    def reset(self) -> None:
```

Comment:
and again. And that goes for all remaining instances which I am not going to comment on individually.

Reply:
Ditto 

> Adding any annotation indicates "please type-check the body of this function".

Changed for now